hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -4119,6 +4119,14 @@
     Mover, and StoragePolicySatisfier.
   </description>
 </property>
+<property>
+  <name>dfs.namenode.get-blocks.check.operation</name>
+  <value>true</value>
+  <description>
+    Whether enable checkOperation when call getBlocks.
+    It is enabled (true) by default.
```

Comment:
It almost look good to me. Just one nit, add "Set false to disable checkOperation and getBlocks for Balancer will route to Standby NameNode for HA mode setup." will be more readable for end user.

Reply:
Thank you for your review.  I have  adjust this property's description and  use this `dfs.namenode.get-blocks.check.operation` property instead of `dfs.ha.allow.stale.reads`  in  NameNodeConnector.java for Balancer.